infra/modules/aws-psoxy-bulk/migration_legacy.tf

```diff
@@ -0,0 +1,27 @@
+
+moved {
+  from = aws_iam_policy.output_bucket_write_policy
+  to   = aws_iam_policy.sanitized_bucket_write_policy
+}
+moved {
+  from = aws_iam_role_policy_attachment.write_policy_for_output_bucket
+  to   = aws_iam_role_policy_attachment.write_policy_for_sanitized_bucket
+}
+moved {
+  from = aws_iam_policy.output_bucket_read
+  to   = aws_iam_policy.sanitized_bucket_read
+}
+
+moved {
+  from = aws_s3_bucket.output
+  to   = aws_s3_bucket.sanitized
+}
+
+moved {
+  from = aws_s3_bucket_server_side_encryption_configuration.output
```

Comment:
If `aws_s3_bucket_server_side_encryption_configuration` is destroyed, do we need to keep the movement from output to sanitized on the migration legacy ?